src/browser/context-menu-template.js

```diff
@@ -0,0 +1,99 @@
+var reload = function(item, focusedWindow) {
+  if (focusedWindow) {
+    focusedWindow.webContents.reload();
+  }
+};
+
+var windows_linux_template = [
+  {
+    label: 'Undo',
+    accelerator: 'CmdOrCtrl+Z',
+    role: 'undo:'
+  },
+  {
+    label: 'Redo',
+    accelerator: 'Shift+CmdOrCtrl+Z',
+    role: 'redo:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    role: 'cut:'
+  },
+  {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    role: 'copy:'
+  },
+  {
+    label: 'Paste',
+    accelerator: 'CmdOrCtrl+V',
+    role: 'paste:'
+  },
+  {
+    label: 'Select All',
+    accelerator: 'CmdOrCtrl+A',
+    role: 'selectAll:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Reload',
+    accelerator: 'CmdOrCtrl+R',
+    click: reload
+  }
+];
+
+var osx_template = [
+  {
+    label: 'Undo',
+    accelerator: 'CmdOrCtrl+Z',
+    selector: 'undo:'
+  },
+  {
+    label: 'Redo',
+    accelerator: 'Shift+CmdOrCtrl+Z',
+    selector: 'redo:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    selector: 'cut:'
+  },
+  {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    selector: 'copy:'
```

Comment:
So it appears that cut/copy/paste all have default actions, so we don't need to code anything?